_ont/ont-leox-lxt-010s-h.md

```diff
@@ -12,27 +12,65 @@ parent: Leox
 | Vendor      | Leox                                 |
 | Model       | LXT-010S-H                           |
 | Chipset     | RTL9601CI                            |
-| Flash       |                                      |
-| RAM         |                                      |
+| Flash       | 128MB                                |
+| RAM         | 32MB                                 |
 | System      | Linux 3.18 (Luna SDK 3.3)             |
 | HSGMII      | ✅                                   |
-| Optics      |                                      |
+| Optics      | SC/UPC                               |
 | IP address  | 192.168.100.1/24                     |
-| Web Gui     |                                      |
+| Web Gui     | ✅                                   |
 | Telnet      | ✅ user `leox`, password `leolabs_7` |
 | Form Factor | miniONT SFP                          |
 
 
 ## List of software versions
-## List of partitions
-## List of firmwares and files
+# List of partitions
```

Comment:
List of partition are (in all other pages) a second level header, use `##` inside `#`.